subtests/docker_cli/systemd/systemd.py

```diff
@@ -0,0 +1,80 @@
+"""
+Summary
+-------
+
+Operational Summary
+---------------------
+    1. Copy Unit file
+    2. Build image using predefined Dockerfile & p4321-server.py
+    3. Run image
+    4. Confirm p4321-server script is running
+    5. Stop container
+    6. cleanup
+
+"""
+
+import shutil
+import time
+from os.path import join
+from autotest.client import utils
+from autotest.client.shared.utils import is_port_free
+from dockertest.subtest import Subtest
+from dockertest.dockercmd import DockerCmd
+
+
+class systemd(Subtest):
+
+    def initialize(self):
+        super(systemd, self).initialize()
+        self.stuff['tcp_ip'] = self.config['tcp_ip']
+        self.stuff['tcp_port'] = self.config['tcp_port']
+        self.stuff['sysd_unit_file'] = self.config['sysd_unit_file']
+        self.stuff['sysd_unitf_dest'] = self.config['sysd_unitf_dest']
+        self.stuff['name'] = self.config['name']
+        self.stuff['self.tmpdir'] = self.tmpdir
```

Comment:
I guess I wasn't very clear. `self.stuff` and `.sub_stuff` are for things that are defined in one place and used in another. The lines above are just duplication, because the values in `.stuff` are identical to the ones in `.config`. 

Put another way, this is a useful definition:

    self.stuff['cmd'] = DockerCmd(....something)

See that DockerCmd returns an actual variable, something changing, that needs to be used later in a different function. But this is not useful:

    self.stuff['a'] = self.config['a']

..because all functions in this module have access to self.config['a']. See the difference?

So, you can just delete lines 29-34.